frontend/src/app/tasks/page.tsx

```diff
@@ -27,7 +27,7 @@ export default async function Tasks() {
       projects={projects}
       taskTypes={taskTypes}
       templates={templates}
-      taskList={<TaskList />}
+      taskList={<TaskList projects={projects} taskTypes={taskTypes} />}
```

Comment:
I have an architectural question here, or it could be a confusion of names. Since a form would be concerned with just one item, why pass the list through? Would it make more sense to have a DayView outer component/container, then a form on the left that gets only the data it needs and a list component on the right that gets the list? 

My overall concern is that when we start building the week/month views, we could find our current structure not as flexible or composable as we need it to be.

Reply:
I think that naming of the component could be improved because it has 4 elements: Select Template, Start Timer button, the Create Task Form and the TaskList, since the first 3 all interact with the form I decided to make the TaskList just a prop so it could be tested in isolation from the form.

the current html structure is like this:
```tsx
<Container display="flex">
  <div>
    <SelectTemplate>
    <StartTimer BUtton>
  </div>
  
  <Divider>
  
  <div>
    <NewTaskForm>
    <TaskList>
  </div>
</Container>
```

We have 3 sections there: 1 top bar that interacts with the form, 2 a divider, 3 the new task form and the task list.  but I think its possible to solve this with a grid layout and make something like this:


But I think its possible to make something like:

```ts
<Container display="grid">
  <SelectTemplate>
  <StartTimerButton>
  <Divider>
  <NewTaskForm>
  <TaskList>
</Container>
```

and then we can wrapp the template select, start timer button and new TaskForm into 1 component and have something like this in the end:

```tsx
const DayView = () => {

  return (
    <Container display="grid">
      <CreateTaskForm />
      
      <Divider gridArea="middle" />
      
      <TaskList gridArea="bottom-right" />
    </Container>
  )
}

const CreateTaskForm = () => {
  return (
    <>
      <SelectTemplate gridArea="top-left" />
      <StartTimeButton gridArea="top-right" />
      <form gridArea="bottom-left" />
    </>
}
```

This way every component would be on its own place :thinking:

Reply:
Thanks for pointing it out @dmtrek14! I will improve that structure

Reply:
as soon as I finish this PR I will start that restructure

Reply:
Sounds good @negreirosleo - thanks for indulging me!